src/Input/Select/Select.tsx

```diff
@@ -209,10 +209,11 @@ class Select extends React.Component<Props, State> {
       if (!activeElement) {
         return;
       }
-
-      const activeElementIndex = Number(get(activeElement, 'dataset.id'));
-      const activeChild = React.Children.toArray(children)[activeElementIndex];
```

Comment:
I think we can keep current implementation (active index), just need to replace `React.Children.toArray(children)` with `filterValue`.

Reply:
Whoo awesome. Thanks @ross!